doc/internal/ReleaseInstructions.md

```diff
@@ -235,19 +292,29 @@ And finally, click on `Publish release`.
 
 ![alt text](.images/release-04.png)
 
-### Post Release Steps
-* Announce new release in Vitess Slack `#general` channel. 
-* Create a new [PR](https://github.com/vitessio/website/pull/670) for Vitess Blog. 
-* Netlify -> Sites →vitess.io → Deploy ![GitHub Post Release01](/doc/internal/.images/post-release-01.png)
-* Coordinate CNCF cross-posting Vitess Blog. 
-* Schedule and publish Tweet on Vitess account.
-* Run following script to once the `base` Docker image is live. 
+### How To Code Freeze
+
+In this example we are going to do a code freeze on the `release-15.0` branch. If we are doing a release candidate, there won't be such branch, hence we need to create it.
```

Comment:
```suggestion
In this example we are going to do a code freeze on the `release-15.0` branch. If we are doing a release candidate, there won't be a branch yet, hence we need to create it.
```

Reply:
Should add a step to make sure your local checkout is up-to-date before creating the branch. Basically `git fetch origin` first.